Description: Upstream changes introduced in version 4.12.0-0ubuntu1
 This patch has been created by dpkg-source during the package build.
 Here's the last changelog entry, hopefully it gives details on why
 those changes were made:
 .
 unity (4.12.0-0ubuntu1) oneiric; urgency=low
 .
   * New upstream release.
     - P3: [natty] Cannot click on indicators (LP: #819202)
     - Icons not visible in the launcher or alt-tab switcher (LP: #833905)
     - [dash] Search field in Unity can not support iBus (LP: #663776)
     - [dash] preferred applications are not stored in GConf anylonger
       (LP: #805063)
     - SysTray icons have a white square background in Unity. (LP: #823407)
     - new dash does not fit a normal netbook screen (LP: #825370)
     - Dash picks random weird colors (yellow, green), fonts invisible (with
       blur en- or disabled) (LP: #828363)
     - Music Lens doesn't display items until you search (LP: #824892)
     - Dash - Adjustments to stateful behaviour  (LP: #838667)
     - Dragging from dash to launcher still doesn't work (LP: #824833)
     - Click on 'More Apps' et al closes the dash (LP: #824842)
     - cannot paste into dash (particularly relevant for alt+f2) (LP: #736222)
     - Ctrl X doesn't cut selected text in Dash search field (LP: #737731)
     - Ctrl+C doesn't copy selected text in Dash search field (LP: #737732)
     - The indicators shows a big white square when systray whitelisted
       (LP: #822672)
     - ARM FTBFS fix (LP: #834576)
   * debian/control:
     - build on latest nux
   * debian/rules:
     - bump shlibs
 .
 The person named in the Author field signed this changelog entry.
Author: Didier Roche <didrocks@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/663776
Bug-Ubuntu: https://bugs.launchpad.net/bugs/736222
Bug-Ubuntu: https://bugs.launchpad.net/bugs/737731
Bug-Ubuntu: https://bugs.launchpad.net/bugs/737732
Bug-Ubuntu: https://bugs.launchpad.net/bugs/805063
Bug-Ubuntu: https://bugs.launchpad.net/bugs/819202
Bug-Ubuntu: https://bugs.launchpad.net/bugs/822672
Bug-Ubuntu: https://bugs.launchpad.net/bugs/823407
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824833
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824842
Bug-Ubuntu: https://bugs.launchpad.net/bugs/824892
Bug-Ubuntu: https://bugs.launchpad.net/bugs/825370
Bug-Ubuntu: https://bugs.launchpad.net/bugs/828363
Bug-Ubuntu: https://bugs.launchpad.net/bugs/833905
Bug-Ubuntu: https://bugs.launchpad.net/bugs/834576
Bug-Ubuntu: https://bugs.launchpad.net/bugs/838667

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- unity-4.12.0.orig/services/CMakeLists.txt
+++ unity-4.12.0/services/CMakeLists.txt
@@ -39,7 +39,7 @@ set(CFLAGS
     "-DGETTEXT_PACKAGE=\"unity\""
     "-DINDICATORDIR=\"${_indicatordir}\""
     "-DINDICATORICONDIR=\"${_iconsdir}\""
-    "-Werror -Wall"
+    "-Wall"
     )
 add_definitions(${CFLAGS})
 
